.github/workflows/build-and-test.yml

```diff
@@ -26,5 +26,7 @@ jobs:
           skip_lint: true
       - name: Lint
         run: bin/hhast-lint
+      - name: Test
+        run: vendor/bin/hacktest tests/
```

Comment:
This is already part of `hhvm/actions/hack-lint-test` above

Reply:
I thought the action means "tests of hack lint".